androidClient/app/src/main/res/layout/content_about_us.xml

```diff
@@ -98,6 +140,33 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/content_about_us_fb_imgbtn" />
 
+    <TextView
+        android:id="@+id/content_about_us_morebymac_tv"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:gravity="center"
+        android:padding="8dp"
+        android:text="More By MAC"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/content_about_us_desc_tv" />
+
+    <android.widget.ListView
+        android:id="@+id/content_about_us_morebymac_lv"
+        android:layout_width="0dp"
+        android:layout_height="250dp"
```

Comment:
Why fixed height?

Reply:
wrap content was not working properly

Reply:
as all members are one below other, you can use linearlayout. (maybe put it inside nested scroll view https://developer.android.com/reference/android/support/v4/widget/NestedScrollView )
So finally something like
```xml
<nestedscrollview>
  <linearlayout>
    <some... views/>
  </linearlayout>
</nestedscrollview>